SQLAlchemy integration fails to parse version `2.0.0rc1`

### How do you use Sentry?

Sentry Saas (sentry.io)

### Version

1.12.1

### Steps to Reproduce

Call `sentry_sdk.init()` with SQLAlchemy integration and install SQLAlchemy==2.0.0rc1.

### Expected Result

no error

### Actual Result

```
      @staticmethod
      def setup_once():
          # type: () -> None
      
          try:
              version = tuple(map(int, SQLALCHEMY_VERSION.split("b")[0].split(".")))
          except (TypeError, ValueError):
  >           raise DidNotEnable(
                  "Unparsable SQLAlchemy version: {}".format(SQLALCHEMY_VERSION)
              )
  E           sentry_sdk.integrations.DidNotEnable: Unparsable SQLAlchemy version: 2.0.0rc1
```

--- a/sentry_sdk/integrations/sqlalchemy.py
+++ b/sentry_sdk/integrations/sqlalchemy.py
@@ -1,5 +1,7 @@
 from __future__ import absolute_import
 
+import re
+
 from sentry_sdk._types import MYPY
 from sentry_sdk.hub import Hub
 from sentry_sdk.integrations import Integration, DidNotEnable
@@ -28,7 +30,9 @@ class SqlalchemyIntegration(Integration):
         # type: () -> None
 
         try:
-            version = tuple(map(int, SQLALCHEMY_VERSION.split("b")[0].split(".")))
+            version = tuple(
+                map(int, re.split("b|rc", SQLALCHEMY_VERSION)[0].split("."))
+            )
         except (TypeError, ValueError):
             raise DidNotEnable(
                 "Unparsable SQLAlchemy version: {}".format(SQLALCHEMY_VERSION)

